docs/integrations/gerrit/locations.md

```diff
@@ -0,0 +1,43 @@
+---
+id: locations
+title: Gerrit Locations
+sidebar_label: Locations
+description: Integrating source code stored in Gerrit into the Backstage catalog
+---
+
+The Gerrit integration supports loading catalog entities from Gerrit hosted gits. Entities can
+be added to [static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+## Configuration
+
+To use this integration, add at least one Gerrit configuration to your root `app-config.yaml`:
+
+```yaml
+integrations:
+  gerrit:
+    - host: gerrit.company.com
+      apiBaseUrl: gerrit.company.com/gerrit
+      gitilesBaseUrl: gerrit.company.com/gitiles
```

Comment:
base URLs typically contain a protocol (making them proper URLs as the name implies) - shouldn't these?

Reply:
Yeah I see in the examples below that that's what seems to be expected

Reply:
Yes, you are correct! I'll fix that and the "microsite" asap!

Reply:
Fixed in #10721 10721

Reply:
Fixed in PR #10721